bravado/http_future.py

```diff
@@ -199,7 +199,7 @@ def response(
 
             swagger_result = self._get_swagger_result(incoming_response)
 
-            if self.operation is None and incoming_response.status_code >= 300:
+            if self.operation is None and incoming_response.status_code >= 400:
```

Comment:
I would not change this behavior (at least on this PR) as users might rely on this.

You're right to mention that RFC7231 does not consider them errors.
As this is independent from the specific issue you're trying to solve (following redirects) I would suggest to create an issue for this and eventually publish a new PR.
NOTE: changing this (so 3xx do not raise exception) requires a major version bump

Reply:
Done. Instead I check the response stored in the exception for the relevant information.

Reply:
Raised #455 to track not raising exceptions when redirects are returned.